### ISSUE TITLE
Application hangs when calling now()

### ISSUE THREAD

**Issue by Kamil-H** [erikc5000/island-time#381]
I'm using the latest version of the library (0.6.3). When I'm trying to obtain a current date with the `Date.now()` application hangs when running it on the iOS Simulator. Is it a known issue? 
I also tried `Instant.now()`, and `DateTime.now()`, and all of them hang the app.

**Comment by erikc5000** [comment]
I'm not aware of such an issue, but 0.6.3 is a little outdated at this point, so I would recommend trying the latest snapshot build. I really need to get a new release.version out though.

**Comment by Kamil-H** [comment]
I was able to reproduce the issue in the simple project here:
https://github.com/Kamil-H/DateIssue
In the common code, I have the following:
```
class Greeting {
    fun greet(): String {
        println("Greet!")
        val now = Date.now()
        println("now: $now")
        return "Now, ${Date.now()}!"
    }
}
```
and on Android, it works fine, the date is printed and displayed on the screen. When I run it on the iOS Simulator with XCode 15.1, the "Greet!" is getting printed, but the application hangs without any message. 

After I switched to the `0.7.0-SNAPSHOT` version, the issue disappeared. 

When can I expect a new version to be released? 🙂
